decentraland/unity-renderer
# fix: textshape for ECS7

## What does this PR change?

This PR fixes the text shape for ECS 7

## How to test the changes?

1.- Download the scene https://github.com/decentraland/ecs7-template (recommended branch: feat/cube-spawner)
2.- Follow the instructions and start the scene with npm start
3.- Use this URL: http://127.0.0.1:8000/?position=0%2C0&ENABLE_ECS7=&renderer-branch=fix%2Ftextshape&kernel-branch=main&SCENE_DEBUG_PANEL=&realm=v1%7E127.0.0.1%3A8000
4.- Change the code to see if everything works as expected


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
